lib/fontcustom/scripts/generate.py

```diff
@@ -126,7 +126,12 @@ def createGlyph( name, source, code ):
     manifest['fonts'].append(fontfile + '.woff')
 
     # Convert EOT for IE7
-    subprocess.call('python ' + scriptPath + '/eotlitetool.py ' + fontfile + '.ttf -o ' + fontfile + '.eot', shell=True)
+    command_args = scriptPath + '/eotlitetool.py ' + fontfile + '.ttf -o ' + fontfile + '.eot'
+    process = subprocess.run('python ' + command_args, shell=True)
+    # Catch command not found error for python vs python3
+    if process.returncode == 127:
+        subprocess.run('python3 ' + command_args, shell=True, check=True)
```

Comment:
Would adding `check=True` to the python2 call and logging the error help with debugging?

Reply:
I'll be honest and say I dont know how to capture errors in python and didnt spend the time to learn so instead went with only adding it here and catching the returncode. If you want I am happy to go and learn, I was just being a bit quick

Reply:
I think it's fine to leave as is then!